streams/src/test/java/org/apache/kafka/streams/integration/SuppressionIntegrationTest.java

```diff
@@ -313,6 +316,124 @@ public void shouldShutdownWhenBytesConstraintIsViolated() throws InterruptedExce
         }
     }
 
+    @Test
+    public void shouldWithLoggingEnable() {
```

Comment:
```suggestion
    public void shouldAllowOverridingChangelogConfig() {
```
Not a big deal, but it's a little nicer when the test methods explain exactly what they are testing.